pkg/util/utils.go

```diff
@@ -37,6 +40,8 @@ type RetriableError struct {
 	Err error
 }
 
+var VERSION semver.Version = semver.MustParse("0.0.0")
```

Comment:
Ah good call. Seems a little cleaner with a setter. Updated to init with a nil pointer